src/components/main-view/main-view.jsx

```diff
@@ -56,12 +57,29 @@ export default class MainView extends React.Component {
 
     return (
       <div className="main-view">
-        {selectedMovie
-          ? <MovieView movie={selectedMovie} onBackClick={newSelectedMovie => { this.setSelectedMovie(newSelectedMovie); }}/>
-          : movies.map(movie => (
-            <MovieCard key={movie._id} movie={movie} onMovieClick={(newSelectedMovie) => { this.setSelectedMovie(newSelectedMovie) }}/>
-          ))
-        }
+        <Navbar className="mb-5" bg="primary" variant="dark">
+          <Navbar.Brand href="#">myFlix</Navbar.Brand>
+          <Nav className="me-auto">
+            <Nav.Link href="#movies">Movies</Nav.Link>
+            <Nav.Link href="#profile">Profile</Nav.Link>
+            <Nav.Link href="#logout">Logout</Nav.Link>
+          </Nav>
+        </Navbar>
+
+        <Row className="main-view justify-content-md-center">
+          {selectedMovie
+            ? (
+              <Col md={8} lg={4}>
+                <MovieView movie={selectedMovie} onBackClick={newSelectedMovie => { this.setSelectedMovie(newSelectedMovie); }}/>
```

Comment:
Ignore this for now but we can fix it in the right exercise. As this one is purely for bootstrap
I don't think when you are viewing a single movie and you click back, we want to set the movie, shouldn't we be removing the set movie or even yet routing to the `history.goBack()`.